### ISSUE TITLE
The SET lost param name

### ISSUE THREAD

**Issue by hiowenluke** [PejmanNik/sqlops-spexecutesql-to-sql#1]
Hi Pejman, 

Source:
``` sql
exec sp_executesql N'update mytable set field=@P1',N'@P1 varchar(10)', 'value'
```

After converted:
``` sql
DECLARE @P1 varchar(10)
SET 'value' --<<-- error

update mytable set field=@P1
```

Please fix it, thanks.

Best,
Owen

**Comment by PejmanNik** [comment]
Hi @hiowenluke 
Thank you for using this extension 🤞
This has happened due to your query parameters value. They are unnamed unlike what I was always seeing.

Fixed in v 0.0.2

**Comment by hiowenluke** [comment]
Hi Pejman,

Thanks a lot.

Best
Owen
